controls/docker_level2.rb

```diff
@@ -132,16 +148,61 @@
   impact 1.0
   title 'Disable operations on legacy registry (v1)'
   desc 'The latest Docker registry is v2. All operations on the legacy registry version (v1) should be restricted.'
-  ref 'https://docs.docker.com/engine/reference/commandline/daemon/'
-  ref 'https://github.com/docker/docker/issues/8093'
-  ref 'https://github.com/docker/docker/issues/9015'
-  ref 'https://github.com/docker/docker-registry/issues/612'
+  tag 'daemon'
+  tag cis: '2.13'
+  ref 'Docker daemon storage driver options', url: 'https://docs.docker.com/engine/reference/commandline/daemon/#storage-driver-options'
+  ref 'Proposal: Provenance step 1 - Transform images for validation and verification', url: 'https://github.com/docker/docker/issues/8093'
+  ref 'Proposal: JSON Registry API V2.1', url: 'https://github.com/docker/docker/issues/9015'
+  ref 'Registry next generation', url: 'https://github.com/docker/docker-registry/issues/612'
+  ref 'Docker Registry HTTP API V2', url: 'https://docs.docker.com/registry/spec/api/'
+  ref 'Creating Private Docker Registry 2.0 with Token Authentication Service', url: 'https://the.binbashtheory.com/creating-private-docker-registry-2-0-with-token-authentication-service/'
+  ref 'New Tool to Migrate From V1 Registry to Docker Trusted Registry or V2 Open Source Registry', url: 'https://blog.docker.com/2015/07/new-tool-v1-registry-docker-trusted-registry-v2-open-source/'
+  ref 'Docker Registry V2', url: 'https://www.slideshare.net/Docker/docker-registry-v2'
 
   describe json('/etc/docker/daemon.json') do
     its(['disable-legacy-registry']) { should eq(true) }
   end
 end
 
+control 'cis-docker-benchmark-2.14' do
+  impact 1.0
+  title 'Enable live restore'
+  desc 'The \'--live-restore\' enables full support of daemon-less containers in docker. It ensures that docker does not stop containers on shutdown or restore and properly reconnects to the container when restarted.'
+  tag 'daemon'
+  tag cis: '2.14'
+  ref 'Add --live-restore flag', url: 'https://github.com/docker/docker/pull/23213'
+
+  describe json('/etc/docker/daemon.json') do
+    its(['live-restore']) { should eq(true) }
+  end
+end
+
+control 'cis-docker-benchmark-2.15' do
```

Comment:
yes i agree, but we should include this in the 2.0 release. i opened a issue #21